RecoTracker/FinalTrackSelectors/src/ES_Tf.cc

```diff
@@ -0,0 +1,3 @@
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+#include "FWCore/Utilities/interface/typelookup.h"
+TYPELOOKUP_DATA_REG(tensorflow::Session);
```

Comment:
Per https://github.com/cms-sw/cmssw/pull/31682#discussion_r505550386, this should be unnecessary and should be removed.